### PR TITLE
[#3] [Flutter] [UI] As a user, I can see the Order Center tab screen

### LINKED ISSUE #3: [Flutter] [UI] As a user, I can see the Order Center tab screen
## Why

We need to implement a simple UI for the new Order Center screen to show all arrived orders from the backend via MQTT.

## Acceptance Criteria

- Implement a list of order items that have:
  - Order id (number).
  - Date time (in `dd/MM/yyyy` format).
  - Customer name
  - Price + `$`
  - 2 buttons: Accept (as primary) & Cancel (as secondary).

## Design

No particular design for this screen. Please try to use this mockup and make it match our Figma design.

image

Figma: https://www.figma.com/file/QQJEM3O8LmntPcU8UrpXOU/POS-iOS-application?type=design&node-id=452%3A532&mode=dev

### PR DESCRIPTION
**Pull request by ryan-conway**
- Close #3 

## What happened 👀

- Created `OrderCenterScreen`
- Added `mockOrders` to `MainViewModel` until integration is finished

## Insight 📝

- UI on `OrderCenterScreen` seemed weird without the header, so I elected to move it outside of the product section widget 🙏🏻 

## Proof Of Work 📹

<image width=600 src="https://github.com/nimblehq/flutter_mqtt_showcase/assets/53168251/8dc34605-e198-4970-ba38-3bba6401b949" />

### CODE REVIEW

File: lib/screen/main/main_screen.dart [hunk 1]
```diff
@@ -204,14 +205,8 @@ class _MainScreenState extends ConsumerState<MainScreen> {
 
   // TODO: Create the notification screen
```
Comment (luongvo) [inline lib/screen/main/main_screen.dart:206]: we can now remove this comment

File: lib/screen/order/order_center_item.dart [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                  item.id,
+                  style: context.textTheme.bodyMedium?.copyWith(
+                      // fontSize: Dimens.fontSize11,
+                      ),
+                ),
+                const SizedBox(height: Dimens.space4),
+                Text(
+                  "\$${item.price}",
+                  style: context.textTheme.bodySmall?.copyWith(
+                      // fontSize: Dimens.fontSize11,
```
Comment (luongvo) [inline lib/screen/order/order_center_item.dart:48]: please remove wip code. The pow looks fine to me 🙏

File: lib/screen/order/order_center_screen.dart [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  Widget build(BuildContext context) {
+    return Container(
+      color: ColorName.whiteSmokeColor,
+      child: Padding(
+        padding: const EdgeInsets.only(
+          top: Dimens.space16,
+          bottom: Dimens.space8,
+          left: Dimens.space16,
+          right: Dimens.space16,
+        ),
```
Comment (luongvo) [inline lib/screen/order/order_center_screen.dart:27]: all sides padding could be 16

**Review by luongvo — COMMENTED** [review]
@ryan-conway The rest lgtm

File: lib/screen/main/main_screen.dart (hunk 1 above)
Comment (luongvo) [reply lib/screen/main/main_screen.dart:206]: Removed in https://github.com/nimblehq/flutter_mqtt_showcase/pull/8/files#diff-a79f88e0db0229c6dfd25fb2b0ce1599b066acedfdfd6035e669f8dc7be4f925L206

File: lib/screen/order/order_center_item.dart [hunk 4]
```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:showcase_flutter_build_web_desktop/extension/context_extension.dart';
+import 'package:showcase_flutter_build_web_desktop/gen/colors.gen.dart';
+import 'package:showcase_flutter_build_web_desktop/model/order_model.dart';
+import 'package:showcase_flutter_build_web_desktop/theme/dimen.dart';
+
+class OrderCenterItem extends StatelessWidget {
+  final OrderModel item;
+  final Function() onAccept;
+  final Function() onReject;
```
Comment (hoangnguyen92dn) [inline lib/screen/order/order_center_item.dart:10]: Could be replaced by VoidCallback

image

File: lib/l10n/app_en.arb [hunk 5]
```diff
@@ -28,5 +28,7 @@
     }
   },
   "tab_checkout": "Checkout",
-  "tab_notifications": "Notifications"
+  "tab_notifications": "Notifications",
+  "button_accept" : "Accept",
+  "button_reject" : "Reject"
```
Comment (hoangnguyen92dn) [inline lib/l10n/app_en.arb:33]: Just a minor, seems the team doesn't follow the convention `type_text_xxx` so this looks weird.

```suggestion
  "accept_text" : "Accept",
  "reject_text" : "Reject"
```

File: lib/model/order_model.dart [hunk 6]
```diff
@@ -0,0 +1,13 @@
+class OrderModel {
+  final String id;
+  final String date;
+  final String customerName;
+  final String price;
+
+  OrderModel({
```
Comment (hoangnguyen92dn) [inline lib/model/order_model.dart:7]:
```suggestion
  const OrderModel({
```

**Review by luongvo — APPROVED** [review]

**Pull request merged**